torch/_guards.py

```diff
@@ -181,6 +184,14 @@ def inner_create_fn(self):
         else:
             return self.create_fn
 
+    @property
+    def name(self):
+        return self.originating_source.name()
+
+    @property
+    def source(self):
```

Comment:
this is a little confusing, and my fault, can you type annotate it `-> GuardSource:` ?

Reply:
Yup, doing it.